RockLib.Configuration.AspNetCore/RockLib.Configuration.AspNetCore.csproj

```diff
@@ -41,14 +36,14 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="RockLib.Configuration" Version="2.5.3" />
+    <PackageReference Include="RockLib.Configuration" Version="3.0.0" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)'=='net5.0'">
+  <ItemGroup Condition="'$(TargetFramework)'!='net48'">
```

Comment:
These two entries with the `Condition` criteria....I'm not sure if we need the conditions for them, or if we even need these references at all anymore. I think I saw something similar in another solution and I just removed both references and everything worked.

Reply:
Removing both references causes build errors as the [IWebHostBuilder Interface](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.hosting.iwebhostbuilder?view=aspnetcore-6.0) needs it.  However, net48 doesn't recognize the reference, hence the second reference/condition.  However, looking at the docs some more I realized I can use just the second reference, dropping the condition, and it builds for all three supported frameworks.